operators/src/upgrade/helm/client.rs

```diff
@@ -255,8 +279,7 @@ impl HelmClient {
             .with_namespace(Some(self.chart.namespace.clone()))
             .get_values()?;
 
-        let _x = String::from_utf8(output.stdout)
-            .map_err(|_| Error::Utf8("Unable to parse values into string".to_string()))?;
+        let _x = String::from_utf8(output.stdout).map_err(|error| Error::Utf8 { source: error })?;
```

Comment:
Why do we need to check conversion here if we are not returning or using the values? How are we getting the values at the caller end?

Reply:
This is not a final code, we are going to save the output in a new file and use it in upgrade.